### DISCUSSION TITLE
How to update the repo in my cli?

### DISCUSSION THREAD

**Discussion by agentjacker** [projectdiscovery/nuclei-templates#6714] [Q&A] [answered]
I cloned the nuclei repo 4 months ago but there are new things added to it so how can i update all this info in the files on my computer. i tried cloning again but it said this file already exists

**Comment by olearycrew** [comment, marked as answer]
You can run `nuclei -ut` to update your templates.

Or if you cloned the templates yourself, you can run `git pull`
